test/test_unistd.rs

```diff
@@ -78,8 +79,8 @@ fn test_mkstemp() {
 
 #[test]
 fn test_getpid() {
-    let pid = getpid();
-    let ppid = getppid();
+    let pid: ::libc::pid_t = getpid().into();
```

Comment:
Why doesn't `getpid()` return a `Result` instead of a `Pid` regardless of success? That would be much more ergonomic then making the user have to cast into a `libc` type just to check if the return value is valid.